scripts/zones/Behemoths_Dominion/mobs/King_Behemoth.lua

```diff
@@ -15,6 +15,18 @@ function onMobInitialize(mob)
     mob:setMobMod(MOBMOD_MAGIC_COOL, 60);
 end;
 
+-----------------------------------
+-- onAdditionalEffect Action
+-----------------------------------
+function onAdditionalEffect(mob,target,damage)
+    if ((math.random(1,100) >= 81) or (target:hasStatusEffect(EFFECT_STUN) == true)) then

```

Comment:
`math.random(0,99) >= 20` Because you are returning zero's if this condition is true (alternatively you can flip your >= to <= but this is more intuitive: 20% chance is what you intended, right?)

`applyResistanceAddEffect(player,target,ELE_LIGHTNING, EFFECT_STUN)` will check the resistance and returns the resistance value with 1.0 being fully unresisted. I'd assume a player has to full resist the effect for it to not land, and odds are the estimated proc rate was while wearing no stun resist gear.